### ISSUE TITLE
Error compiling with rebar3

### ISSUE THREAD

**Issue by yokomizor** [processone/stun#20]
Hi,

Running `rebar3 compile` causes this error:

```
===> Verifying dependencies...
===> Compiling stun
===> Compiling /Users/rogerioyokomizo/projects/stun/c_src/*.c
===> clang: error: no such file or directory: 'c_src/*.c'
clang: error: no input files
```

Turning provider_hooks off solves the problem:
https://github.com/processone/stun/blob/master/rebar.config.script#L106
```
- {[provider_hooks], IsRebar3,
-	  AppendList([{pre, [
-			     {compile, {pc, compile}},
-			     {clean, {pc, clean}}
-			    ]}]), []},
```

**Comment by yokomizor** [comment]
Possible workaround: https://github.com/erlio/vernemq/pull/700/files

**Comment by prefiks** [comment]
Ups, you are right, must have left that when i ported rebar.config from different project.

**Comment by yokomizor** [comment]
Looks like it is a port_compiler problem.

**Comment by prefiks** [comment]
This is problem in our rebar.config.script, i try to call those hooks, without loading pc plugin.

**Comment by yokomizor** [comment]
The plugin is being loaded I guess: https://github.com/processone/stun/blob/87cafb5fc84fbed97380a5c267610fb1e7ad7488/rebar.config.script#L103

**Comment by prefiks** [comment]
Fixed by commit: 6a07ea2ad5a1149585956fc85d54dbae7a5d0f3b
